### ISSUE TITLE
[swag] Requesting one last MicroProfile t-shirt, hoodie, whatever...

### ISSUE THREAD

**Issue by kwsutter** [microprofile/microprofile-marketing#631]
Due to the lack of in-person conferences or in-person anything the last two years, I'm wondering if I could request one last *something* wearable from MicroProfile marketing...  :-)  Please!  Thank you!

**Comment by aeiras** [comment]
Of course, Kevin. 

Count on it. I had you in my list to send MP stuff this year. 
However, you already have the old shirts. :) No other items are available given the lack of travel or need.

We could send you maybe 2 MP shirts from 2021 print (not the current design), would that work for you?
MP discussed on Monday its desire to distribute Dec 2021 t-shirt order in Q1 2022. You will get that one too. :)

**Comment by kwsutter** [comment]
You know the phrase...  Beggars can't be choosers!  :-)  I would just like one last memento of the MicroProfile community.  Thank you!

**Comment by aeiras** [comment]
Kevin, 

I like that phrase, yet it will never apply to you in anything related to MP. :) You are an MP co-founder, formidable Steward, and Maintainer for +5 & 1/2 years. Those are all merits based on your actions. 

Let's keep this ticket open to get you the updated MPWG 2021 shirt in Q1 2022. 
 
On the side, I will also send you the old t-shirt (you have it, but replacements are OK :) later this month, sadly no hoodies.

**Comment by aeiras** [comment]
Hola and happy and healthy 2022 dear Kevin!!!

No shipment yet as covid numbers are too great in my city to venture out however, given the delay, my hope is that went things slow down, we can send you also the 2021 shirt version. 
Thank you for the patience, Fuentes abrazos

**Comment by aeiras** [comment]
Hola @kwsutter 

2021 shirts printed was halted but it will resume soon and once that is ready, I will make sure you get your swag!  ETA end of Feb if newest design is approved by the EF :)

**Comment by aeiras** [comment]
Kevin, 

I hope this finds you and yours happy and warm! :) 

We are still awaiting for the 2021 swag order. Please check status in this ticket if wondering...  https://github.com/eclipse/microprofile-marketing/issues/633

As soon as t-shirts are available, yours will be prioritized. :) Hugs!

**Comment by aeiras** [comment]
Hola Kevin, 

Shirts have been finally printed, yeayy!

Expect fun update on the shippment in May :)

**Comment by aeiras** [comment]
@@kwsutter, first miss you lots. I hope you are having a great summer, Kevin :) 

Finally, this week during 2 calls, the swag delays have been crushed and MP can move forward shipping the 380 shirts from 2021 order that have been on hold at the printing Vendor. Yupiee!

**Comment by m-reza-rahman** [comment]
This is sorted. I will close this issue when the swag is sent to Kevin from IBM. @Emily-Jiang is aware of this.

**Comment by m-reza-rahman** [comment]
Kevin has received his swag.
